src/modules/Dashboard/components/ActivityGrid/ActivityGrid.utils.tsx

```diff
@@ -44,7 +42,7 @@ export const getActivityActions = ({
       action: exportData,
       title: t('exportData'),
       context: { appletId, activityId },
-      isDisplayed: true,
+      isDisplayed: canAccessData,
```

Comment:
In `FlowGrid.hooks.tsx`, the Export Data menu item has `disabled: true` at all times, not yet sure why. Presumably because it's unimplemented? However, here, it's not disabled, but when it's available (i.e. when `canAccessData == true`), it just shows a TODO alert.

Do you know if this action is expected to be implemented by launch? In any case, it seems that `ActivityGrid` and `FlowGrid` should treat this menu item the same…

Reply:
Not sure why `disabled: true` was added but also thought it was odd, maybe @JustinNusca can chime in.

My guess is it should be enabled as Data Export is supposed to be part of R1, but unsure if Flows are able to be exported the same way Activities are currently handled.

Reply:
Another thing, yes this functionality is meant for R1 and removing the TODO alerts will be covered by [M2-6736](https://mindlogger.atlassian.net/browse/M2-6736)

Reply:
I meant to reply earlier to speak to this — yeah, I set `disabled: true` initially.

At the time, it seemed like we were in the practice of adding a lot of placeholder UI for functionality that wasn't yet present. I opted to just set `disabled`: it felt less confusing/disruptive from the user perspective than having an enabled-but-nonfunctional button and/or alert prompts, with the expectation that the presence of the menu item would be clear enough for whoever eventually needed to wire it up.

In the end, it seems like all these "Export Data" options had the requisite functionality for supporting these options all along, and yeah, there's a couple tasks currently scheduled for Sprint 10 that should handle them.